persistence/msmq/subscription_config_msmqtransport_[1,).partial.md

```diff
@@ -2,11 +2,13 @@ To configure MSMQ as the subscription persistence:
 
 snippet: ConfiguringMsmqPersistence
 
-By default NServiceBus uses a queue called `NServiceBus.Subscriptions`. If not specified otherwise, all endpoints will share that queue to store subscriptions. The following warning message will get logged:
+If the subscription storage is not specified, NServiceBus uses a queue called `[Endpoint].Subscriptions`. This is to avoid all endpoints deployed on the same server from having to share the same queue to store subscriptions. When a subscription queue is not specified and the default queue is being used, the following message will get logged:
 
-> NServiceBus.Features.MsmqSubscriptionPersistence Could not find configuration section for Msmq Subscription Storage and no name was specified for this endpoint. Going to default the subscription queue
+>The queue used to store subscriptions has not been configured, the default '{EndpointName}.Subscriptions' will be used.
 
-WARNING: When using MSMQ Subscription Persistence on multiple endpoints running on the same machine, every endpoint **must have** a dedicated subscription storage queue.
+WARNING: When using MSMQ Subscription Persistence on multiple endpoints running on the same machine, every endpoint **must have** a dedicated subscription storage queue. 
```

Comment:
Make this an INFO with an *should have* instead? I created the doco based on discussions on Slack but I've never been able to actually create subscription loss.

Reply:
@ramonsmits - Not sure why this is showing up as changed. But the warning text is part of the original docs. IMO, I don't think we should downgrade to INFO. Be happy to discuss this further.